wpiutil/src/main/native/thirdparty/llvm/include/wpi/MathExtras.h

```diff
@@ -180,8 +203,10 @@ inline uint64_t maxUIntN(uint64_t N) {
 
 /// Gets the minimum value for a N-bit signed integer.
 inline int64_t minIntN(int64_t N) {
-  assert(N > 0 && N <= 64 && "integer width out of range");
+  assert(N <= 64 && "integer width out of range");
```

Comment:
This needs the `> 0` check, as N is signed.

Reply:
`>= 0` should be sufficient since the function checks for 0.